portfolio/src/main/webapp/css/projects.css

```diff
@@ -0,0 +1,66 @@
+.blackBackgroundImage, .foodDesertImage, .roomrImage {
```

Comment:
Same comments as the other file.

It might be worth it for us to pull some of this out into a "common.css" that is imported on every page, if there's a lot of duplicated styles.

Reply:
good idea